app/scripts/models/auth_brokers/fx-ios-v1.js

```diff
@@ -11,19 +11,89 @@ define(function (require, exports, module) {
   'use strict';
 
   const _ = require('underscore');
-  const FxDesktopV1AuthenticationBroker = require('../auth_brokers/fx-desktop-v1');
+  const FxDesktopChannel = require('../../lib/channels/fx-desktop-v1');
+  const FxSyncChannelAuthenticationBroker = require('../auth_brokers/fx-sync-channel');
+  const HaltBehavior = require('../../views/behaviors/halt');
   const NavigateBehavior = require('../../views/behaviors/navigate');
   const UserAgent = require('../../lib/user-agent');
 
-  const proto = FxDesktopV1AuthenticationBroker.prototype;
+  const proto = FxSyncChannelAuthenticationBroker.prototype;
+
+  const FxiOSV1AuthenticationBroker = FxSyncChannelAuthenticationBroker.extend({
+    type: 'fx-ios-v1',
+
+    commands: {
+      CAN_LINK_ACCOUNT: 'can_link_account',
+      CHANGE_PASSWORD: 'change_password',
+      DELETE_ACCOUNT: 'delete_account',
+      LOADED: 'loaded',
+      LOGIN: 'login'
+    },
+
+    defaultBehaviors: _.extend({}, proto.defaultBehaviors, {
+      // about:accounts displays its own screen after sign in, no need
+      // to show anything.
+      afterForceAuth: new HaltBehavior(),
+      // about:accounts displays its own screen after password reset, no
+      // need to show anything.
+      afterResetPasswordConfirmationPoll: new HaltBehavior(),
+      // about:accounts displays its own screen after sign in, no need
+      // to show anything.
+      afterSignIn: new HaltBehavior(),
+      // about:accounts displays its own screen after sign in, no need
```

Comment:
I was wrong, this comment does need to be updated. A similar comment to lines 46-47 should be put here. Or remove the comment all together

Reply:
> A similar comment to lines 46-47 should be put here.

@shane-tomlinson should they be same or it should it contain the sign in message, like this,

```js
// about:accounts display the "Signin" screen after
// the users verify their email
afterSignInConfirmationPoll: new NavigateBehavior('signin_confirmed'),
```

Reply:
> // about:accounts display the "Signin" screen after
> // the user sign in is successful

How about changing "Signin" to "Signin confirmed", and removing the last "l" in "successfull"

Reply:
> and removing the last "l" in "successfull"

@shane-tomlinson  that was a typo, I think you're talking about adding last 'l' in "sucessfull", am I right ?

Reply:
"successfull" => "successful". I accidentally deleted the last "l" from "full" in the above comment.

Reply:
@shane-tomlinson it happened because first I commented that "users sign in successfully" and then I changed it to this but forgot to correct the spelling of successful.